### PR TITLE
i18n(zh-cn): Update content about upstream #8283

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update `actions-without-server-output-error.mdx`
Update `error-reference.mdx`

#### Related issues & labels (optional)

- #8283

### CODE REVIEW

**Comment by liruifengv** [comment]
把 error-reference 也更新过来吧。我的那个close了

**Comment by huyikai** [comment]
> 把 error-reference 也更新过来吧。我的那个close了

好的

File: src/content/docs/zh-cn/reference/errors/actions-without-server-output-error.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Actions must be used with server output.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **ActionsWithoutServerOutputError**: 启用了 Actions 但未设置服务器构建输出。创建可调用的后端函数需要一个服务器。要将路由部署到服务器，请在你的 astro 配置中添加一个服务器适配器。
+
+## 出了什么问题？
+你的项目必须有服务器输出，才能与 Actions 创建后端函数。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/actions-without-server-output-error.mdx:10]:
```suggestion
你的项目必须有服务器输出，才能用 Actions 创建后端函数。
```

**Review by liruifengv — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**